### ISSUE TITLE
macOS Support

### ISSUE THREAD

**Issue by dhulihan** [rstemmer/id3edit#24]
Installation via `brew` would be very nice and could increase user adoption quite a bit!

**Comment by dhulihan** [comment]
Agreed, it is a good first issue, more of a "wishlist item" from my perspective 😸. Thanks for your consideration!

**Comment by rstemmer** [comment]
Since version 2.2.0 macOS is supported.

The repository can be cloned, the source build on macOS:

```bash
xcode-select --install # If not already done in the past

# Install libprinthex (dependency of id3edit
git clone https://github.com/rstemmer/libprinthex.git
cd libprinthex
./build.sh
sudo ./install.sh # Installs to /usr/local/{lib,include}


# Install id3edit
git clone https://github.com/rstemmer/id3edit.git
cd id3edit
./build.sh
sudo ./install.sh
```

There is also a [Dockerfile](https://github.com/hckurniawan/docker-id3edit) available from [Hendri Kurniawan (hckurniawan)](https://github.com/hckurniawan)

So basically "macOS Support" is done :) Only `brew` support is missing.
